### PR TITLE
Rollup of fixes to iree-check-module.

### PR DESCRIPTION
**Pull request by benvanik**
With this the check module tests now run on Windows.

### CODE REVIEW

File: build_tools/cmake/iree_check_test.cmake [hunk 1]
```diff
@@ -79,18 +79,18 @@ function(iree_check_test)
     iree_modules_check_iree-check-module
   )
 
-  string(REPLACE "_" "/" _PACKAGE_PATH ${_PACKAGE_NAME})                                                                                                    
-  set(_NAME_PATH "${_PACKAGE_PATH}:${_RULE_NAME}") 
+  string(REPLACE "_" "/" _PACKAGE_PATH ${_PACKAGE_NAME})
```
Comment (GMNGeoffrey) [inline build_tools/cmake/iree_check_test.cmake:82]: Did this change it to windows line endings or something? :nauseated_face:  I don't see a diff

File: iree/modules/check/check_module_main.cc [hunk 2]
```diff
@@ -181,8 +186,16 @@ StatusOr<int> Run() {
...
 extern "C" int main(int argc, char** argv) {
   InitializeEnvironment(&argc, &argv);
+  IREE_FORCE_BINARY_STDIN();
+
+  if (argc < 2) {
+    LOG(ERROR)
+        << "A binary module file path to run (or - for stdin) must be passed";
+    return -1;
+  }
+  auto input_file_path = std::string(argv[1]);
```
Comment (GMNGeoffrey) [inline iree/modules/check/check_module_main.cc:196]: Why is changing this to a positional argument necessary? I think whatever changes we make here should also be reflected in iree-run-module and iree-benchmark-module, which currently use the same format (can be a followup). I guess that will be necessary to make them run on Windows anyway?

Comment (benvanik) [reply iree/modules/check/check_module_main.cc:196]: I have a bug tracking the same work for iree-run-mlir (#1281). I don't want proliferation of the bad style as you start to ramp up tests on this - easier to fix now.

File: build_tools/cmake/iree_check_test.cmake (hunk 1 above)
Comment (benvanik) [reply build_tools/cmake/iree_check_test.cmake:82]: you had trailing whitespace

Comment (GMNGeoffrey) [reply build_tools/cmake/iree_check_test.cmake:82]: Ah thanks :-/

File: iree/modules/check/check_module_main.cc (hunk 2 above)
Comment (GMNGeoffrey) [reply iree/modules/check/check_module_main.cc:196]: Ok. Still seems like flag to positional argument is unrelated, but I'm fine with either as long as it's consistent between tools.

**Pull request closed without merging**